doc/architecture/decisions/0005-queue-api.md

```diff
@@ -107,7 +107,7 @@ Returns
       "eventType": "DEATH_NOTIFICATION",
       "sourceId": "{death certificate number}",
       "eventData": {
-        "firstName": "",
+        "firstNames": "",
```

Comment:
I think HMPO use `forenames` are we trying to be consistent with that, or our own schema?

Reply:
We'll also need to tell DWP about this as a breaking change

Reply:
yes, I think we still should do it because its more accurate, what do you think?